.github/workflows/conflict-labeler.yml

```diff
@@ -5,14 +5,15 @@ on:
     branches:
       - master
   pull_request_target:
+    types: [synchronize]
 
 jobs:
   Label:
     if: github.actor != 'PJBot'
     runs-on: ubuntu-latest
     steps:
       - name: Check for Merge Conflicts
-        uses: ike709/actions-label-merge-conflict@9eefdd17e10566023c46d2dc6dc04fcb8ec76142
+        uses: eps1lon/actions-label-merge-conflict@v3.0.0
         with:
```

Comment:
Change is undocumented?

Reply:
Commented change was introduced by 3530b38fa9fa1a3b66dde1982840b602e25ea2a5 . ike709's fork of eps1lon's original github action introduces functionality to skip draft PR's which is a nicety, but this forked action has not been committed to for two years and has their latest version published as `v2.0.1` which does not include the node version upgrade required, hence the change back to the "upstream" version of this action at the cost of the workflow also labelling conflicting draft PR's.

If this functionality is desired, it should be possible to simply define this in the workflow file, not inside of the actual action, which leaves the question, do you want me to look into this?

Reply:
ike attempted to do this forever ago when he first looked into it and determined that it wasnt possible through the workflow file for whatever reason, you can try and do it though, its definitely desired functionality since we have a lot of drafts